internal/cli/terraform_fetcher.go

```diff
@@ -208,6 +212,45 @@ func (f *organizationResourceFetcher) FetchData(ctx context.Context) (importData
 	return data, nil
 }
 
+func (f *resourceServerResourceFetcher) FetchData(ctx context.Context) (importDataList, error) {
+	var data importDataList
+
+	var page int
+	for {
+		apis, err := f.api.ResourceServer.List(
+			ctx,
+			management.Page(page),
+			management.IncludeFields("id", "name"),
+		)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, api := range apis.ResourceServers {
+			if api.GetName() == "Auth0 Management API" {
```

Comment:
**Note:** Management API is intentionally excluded from export.

Reply:
There are tho a few properties that could be configured on the management API 🤔